contracts/contracts/strategies/AaveStrategy.sol

```diff
@@ -269,9 +269,10 @@ contract AaveStrategy is InitializableAbstractStrategy {
         // Collect available rewards and restart the cooldown timer, if either of
         // those should be run.
         if (block.timestamp > windowStart || cooldown == 0) {
+            uint256 assetsLen = assetsMapped.length;
```

Comment:
heh did the same change in one of my branches after the latest slither update :)